### ISSUE TITLE
FormField with Select + Search receives double focus 

### ISSUE THREAD

**Issue by taysea** [grommet/grommet#5887]
### Expected Behavior

When Select search input is active element, it should be the only element with focus indicator.

### Actual Behavior

For certain themes (such as HPE), focus indicator is being shown on FormField and Select search input.

### URL, screen shot, or Codepen exhibiting the issue

![double-focus](https://user-images.githubusercontent.com/12522275/148858331-2975d7dc-722f-405d-a77e-bfe9e537e470.png)

https://design-system.hpe.design/components/select?q=select#select-with-search

### Steps to Reproduce

1. Open Search
2. See double focus.

### Your Environment

- Grommet version: Stable
- Browser Name and version: Chrome Version 97.0.4692.71
- Operating System and version (desktop or mobile): MacOS Big Sur
